src/webapp/components/DeliveryTable.js

```diff
@@ -66,66 +48,46 @@ const DeliveryTable = ({ rows }) => {
     };
   });
 
+  const renderTableHeadRow = () => {
```

Comment:
nit: can you make this a stateless component too? e.g. `const TableHeadRows = ()=> .<TableRow>...</TableRow>` it can stay in this file.  keeping things consistent as components makes code easier to read!